src/Plugin.php

```diff
@@ -63,14 +63,13 @@ public static function wp_enqueue_scripts() {
     wp_enqueue_script('gallerya-script-custom', static::getBaseUrl() . '/dist/scripts/script.min.js', ['gallerya-script-libs'], FALSE, TRUE);
   }
 
-  public static function post_gallery_init() {
+  public static function gallerya_post_init() {
     $url = home_url(add_query_arg(array()));
     $id = url_to_postid($url);
-    $post_type = '';
-    if (get_post_type($id) === $post_type || get_post_type() == '') {
+    $post_type = apply_filters('gallerya_post_init', 'post');
```

Comment:
As the gallery shortcode could be added in almost every wysiwyg editor IMO it would be better to use something like the following to ensure the gallery is modified properly:

```php
$post_type = get_post_type();
$post_types = apply_filters('gallerya/post_types', get_post_types());
if (!isset($post_types[$post_type])) {
  return;
}
```